src/get-started/java-binding.md

```diff
@@ -174,11 +175,11 @@ A set of named collections constitute a *service scheme*. For convenient access
 to service collections you can implement a factory of service collections.
 
 *The state of the service in the blockchain* is determined by the list of
-root hashes of its Merkelized collections. Said root hashes, when aggregated
-with root hashes of other Merkelized collections in the blockchain, form a
+index hashes of its Merkelized collections. Said index hashes, when aggregated
```

Comment:
is determined by the list of hashes. [any hashes — nothing requires to use index hashes] Usually, it is comprised of the hashes of its Merklezied collections. State hashes of each service are aggregated in a single blockchain state hash, ...